src/main/java/net/dries007/tfc/compat/jei/categories/HeatMetalCategory.java

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Can `HeatMetalCategory` and `HeatSimpleCategory` be merged into a generic heat recipe category? It should show the input, and both possible outputs (items and fluids), along with the temperature (in red, yellow, brilliant white, etc.)